src/components/Accordion/Accordion.stories.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 import { Accordion } from './Accordion';
 
 export default {
-  title: 'Organisms/Interactive/Accordion',
+  title: 'Components/Accordion',
```

Comment:
Could remove the `title` from all stories and let Storybook auto-magically figure out the structure.

Tried it previously in #813 and it works. But pretty sure it took some config to make it work well for our case.

Reply:
I don't want to have the components in the root sb folder (which is default without `title`) or have individual folders for each component. Will update the plop stories template to auto place in `Components/` to ease dev experience